modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -135,3 +135,32 @@ grains:
       - name.one
       - name.two
 ----
+
+
+
+== SSL Errors on Registration
```

Comment:
I'd do this more generic as I told before. So people can go here in the event the Apache config is rolled back to allow only TLS 1.2